tests/.infra/crw-ci/nightly/k8s/multi-host/Jenkinsfile

```diff
@@ -0,0 +1,180 @@
+                                    build job: 'basic-MultiUser-Che-check-e2e-tests-against-k8s',
+                                            parameters: [
+                                                    booleanParam(name: 'cheSingleHost',
+                                                        value: cheSingleHost.toBoolean()),
+
+                                                    string(name: 'cheImageRepo',
+                                                            value: 'quay.io/eclipse/che-server'),
+
+                                                    string(name: 'cheImageTag',
+                                                            value: 'nightly'),
+
+                                                    booleanParam(name: 'buildChe',
+                                                            value: false),
+
+                                                    string(name: 'ghprbSourceBranch',
+                                                            value: 'master'),
+
+                                                    string(name: 'ghprbPullId',
+                                                            value: ''),
+
+                                                    string(name: 'e2eTestToRun',
+                                                            value: 'test-git-ssh'),
+
+                                                    string(name: 'testWorkspaceDevfileUrl',
+                                                            value: ''),
+
+                                                    booleanParam(name: 'createTestWorkspace',
+                                                            value: false),
+
+                                                    string(name: 'e2eTestParameters',
+                                                            value: "-e TS_GITHUB_TEST_REPO_ACCESS_TOKEN=$github_oauth_token  -e TS_GITHUB_TEST_REPO=chepullreq4/Spoon-Knife -e NODE_TLS_REJECT_UNAUTHORIZED=0"),
+
+                                                    string(name: 'chectlPackageUrl',
+                                                            value: ''),
+
+                                                    string(name: 'cheE2eImageTag',
+                                                            value: 'nightly')
+                                            ]
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+    }
+
+    post {
+        failure {
+            script {
+                if (emails != "") {
+                    mail to: emails,
+                        subject: "Jenkins Job $currentBuild.fullDisplayName has failed.",
+                        body: "Build $currentBuild.absoluteUrl has failed."
+                }
+            }
+        }
```

Comment:
Could you, please, add code to send the mail in case of successful execution as well:
```
        success {
            script {
                if (emails != "") {
                    mail to: emails,
                            subject: "Jenkins Job $currentBuild.fullDisplayName has passed successfully.",
                            body: "Build $currentBuild.absoluteUrl has passed successfully."
                }
            }
        }
```

Reply:
Done.

Reply:
Thank you!